openmetadata-ui/src/main/resources/ui/src/components/DatePickerMenu/DatePickerMenu.component.tsx

```diff
@@ -35,42 +36,39 @@ import {
 import './date-picker-menu.less';
 
 interface DatePickerMenuProps {
+  defaultDateRange?: Partial<DateRangeObject>;
   showSelectedCustomRange?: boolean;
   handleDateRangeChange: (value: DateRangeObject, days?: number) => void;
   options?: DateFilterType;
-  defaultValue?: string;
   allowCustomRange?: boolean;
   handleSelectedTimeRange?: (value: string) => void;
 }
 
-function DatePickerMenu({
+const DatePickerMenu = ({
```

Comment:
As we are making changes in this common component, it would be good if you check all the places where it is being used.

Reply:
Yes, I have verified the scenarios, and it's working as expected. once this merges I need to make some changes in the collate side.